### PR TITLE
Portofolio mobile skeleton

### PR DESCRIPTION
**Pull request by Arturgouveia1970**
final linters errors

### CODE REVIEW

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @Arturgouveia1970 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## Highlights
- Linters passed ✅ 
- Well described readme ✅ 
- Meaningful commit message ✅ 

## Required Changes ♻️

- [ ] Please apply the changes listed below.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Arturgouveia1970** [comment]
Hello Aime, Would you be so kind as to giving me more precise instructions on the last 3 points. KindlyArturEnviado do meu telemóvel Huawei.-------- Mensagem original --------De: Aime Malaika ***@***.***>Data: quarta-feira, 16 de março de 2022, 22:13Para: Arturgouveia1970/Personal-portofolio ***@***.***>Cc: Arturgouveia1970 ***@***.***>, Mention ***@***.***>Assunto: Re: [Arturgouveia1970/Personal-portofolio] Portofolio mobile skeleton (PR #3)

@aimemalaika requested changes on this pull request.



Hi @Arturgouveia1970 ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!



Highlights

Linters passed ✅Well described readme ✅Meaningful commit message ✅

Required Changes ♻️

 Please apply the changes listed below.

Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.



As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.



In index.html:

> @@ -0,0 +1,34 @@

+<!DOCTYPE html>

+<html lang="en">

+<head>

+    <meta charset="UTF-8">

+    <meta http-equiv="X-UA-Compatible" content="IE=edge">

+    <meta name="viewport" content="width=device-width, initial-scale=1.0">

+    <link rel="stylesheet" href="styles.css">

+    <title>Portofolio</title>

+</head>

+<body>

+  <header>

+    <div class="logo">Artur Gouveia</div>



 Kindly use the a tag for the website logo







In index.html:

> +    <ul>

+      <li><a href="#Portofolio">Portofolio</a></li>

+      <li><a href="#About">About</a></li>

+      <li><a href="#Contact">Contact</a></li> 

+    </ul>



 The hamburger button should be on the right and in a nav tag







In index.html:

> +  <section class="headline">

+    <div class="text">

+        <h1>Hey there. <br>I am Artur</h1>

+        <h4>I am a software Developer</h4>

+        <p>I can help you build a product , feature or website Look<br>

+             through some of my work and experience! If you like what<br> 

+             you see and have a project you need coded,<br> 

+             don’t hestiate to contact me.

+        </p>

+    </div>

+

+  </section>



 Kindly use the appropriate background image for the section, check the Figma design for more description.







In styles.css:

> +.text {

+  padding-left: 125px;

+}



 Kindly reduce the padding to match the text alignment to look like the Figma design. Kindly use flexbox to align the contents.

**Comment by aimemalaika** [comment]
hello @Arturgouveia1970

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <title>Portofolio</title>
+</head>
+<body>
+  <header>
+    <div class="logo">Artur Gouveia</div>
+    <ul>
+      <li><a href="#Portofolio">Portofolio</a></li>
+      <li><a href="#About">About</a></li>
+      <li><a href="#Contact">Contact</a></li> 
+    </ul>
```
[the remark this answers is not in the document]
Comment (Kingstalux) [reply index.html]: resolved 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        <h1>Hey there. <br>I am Artur</h1>
+        <h4>I am a software Developer</h4>
+        <p>I can help you build a product , feature or website Look<br>
+             through some of my work and experience! If you like what<br> 
+             you see and have a project you need coded,<br> 
+             don’t hestiate to contact me.
+        </p>
+    </div>
+
+  </section>
```
[the remark this answers is not in the document]
Comment (Kingstalux) [reply index.html]: resolved 👍

**Review by Kingstalux — APPROVED** [review]
Hi @Arturgouveia1970,
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
- [ ] (OPTIONAL) Kindly reduce the gap between the nav bar and the heading as your is too large 👍 
- Your design:
Annotation 2022-03-17 191113
- Figma design:


Annotation 2022-03-17 191152
- [ ] (OPTIONAL) It would be great if you could use the same icons as used in the template design. Right now your LinkedIn icon is different 
- Your icons:
Annotation 2022-03-17 191210


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Arturgouveia1970** [comment]
Hi guys,
while trying to make a commit got this message: error: failed to push some
refs to 'github.com:Arturgouveia1970/Personal-portofolio.git'.
I am desperate as I have my first weekly deadline.
please help
Artur

**Pull request merged**